specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,794 @@
+                        },
+                        "aznsAction": {
+                          "actionGroup": [],
+                          "emailSubject": "Email Header",
+                          "customWebhookPayload": "{}"
+                        }
+                      }
+                    }
+                  },
+                  {
+                    "id": "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/Rac46PostSwapRG/providers/microsoft.insights/scheduledQueryRules/logalertfoo",
+                    "name": "logalertfoo",
+                    "type": "Microsoft.Insights/scheduledQueryRules",
+                    "location": "West US",
+                    "tags": {
+                      "hidden-link:/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/Rac46PostSwapRG/providers/microsoft.insights/workspaces/servicedeskeus": "Resource"
+                    },
+                    "properties": {
+                      "description": "log alert description",
+                      "Enabled": "true",
+                      "skuType": "L2",
+                      "lastUpdatedTime": "2017-06-23T21:23:52.0221265Z",
+                      "provisioningState": "Succeeded",
+                      "source": {
+                        "query": "Type=HearBeat",
+                        "queryType": "ResultCount",
+                        "datasourceId": "/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/Rac46PostSwapRG/providers/microsoft.insights/workspaces/servicedeskeus"
+                      },
+                      "schedule": {
+                        "frequencyInMinutes": 15,
+                        "timeWindowInMinutes": 15
+                      },
+                      "action": {
+                        "odata.type": "Microsoft.WindowsAzure.Management.Monitoring.Alerts.Models.Microsoft.AppInsights.Nexus.DataContracts.Resources.ScheduledQueryRules.AlertingAction",
+                        "severity": "1",
+                        "status": "Inactive",
+                        "trigger": {
+                          "thresholdOperator": "GreaterThan",
+                          "threshold": 3
+                        },
+                        "aznsAction": {
+                          "actionGroup": [],
+                          "emailSubject": "Email Header",
+                          "customWebhookPayload": "{}"
+                        }
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-odata": "#/definitions/LogSearchRuleResource"
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
```

Comment:
this should also be readonly since the name comes from URL

Reply:
Fixed in latest iteration